test/zencode/w3c.bats

```diff
@@ -701,3 +701,39 @@ EOF
     run $ZENROOM_EXECUTABLE -z -a did_documents.json verify_wrong_did_doc.zen
     assert_line '[W]  The signer id in proof is different from the one in not_signer_did_document'
 }
+@test "JWT HS256 creation" {
+    cat <<EOF > jwt_hs256.data
+{
+	"payload": {
+        "iat": 15162,
+        "name": "John Doe",
+        "sub": "1234567890"
+    },
+	"password": "password"
+}
+EOF
+    cat <<EOF | zexe jwt_hs256.zen jwt_hs256.data
+Scenario 'w3c': did document manipulation
+Given I have a 'string dictionary' named 'payload'
+Given I have a 'string' named 'password'
+
+When I create the jwt hs256 of 'payload' using 'password'
```

Comment:
Are we sure we want to call this `jwt hs256`?

Reply:
hs256 is the cryptographic schema we are using, I could call it only `jwt` and if, in future, we implement other schemas we have to rename it before using another